dfsclient/src/main/java/org/hpccsystems/dfs/client/RowServiceOutputStream.java

```diff
@@ -182,26 +203,64 @@ public class RowServiceOutputStream extends OutputStream
             throw new Exception(errorMessage);
         }
 
+        //------------------------------------------------------------------------------
+        // Check protocol version
+        //------------------------------------------------------------------------------
+
+        try
+        {
+            String msg = makeGetVersionRequest();
```

Comment:
what's the expected return format, and do we log the actual interface version used?

Reply:
At the moment I am not checking the version number, just if we got a valid response.

Reply:
do we know the expected response format? If so, let's document here (as a comment perhaps).
Can we extract the reported interface version? If so, let's capture it and make it available, or do we expect this version to be the last change?

Reply:
Yep, I do. Good point, I will add that